.yamato/shadergraph-linux-openglcore.yml

```diff
@@ -133,6 +139,8 @@ ShaderGraph_Linux_OpenGLCore_editmode_mono_Linear_2021.2:
       -  command: sudo unity-downloader-cli --source-file unity_revision.txt -c editor -c il2cpp  --wait --published-only
          retries: 2
       -  command: DISPLAY=:0.0 ./utr --artifacts_path=TestProjects/ShaderGraph/test-results --editor-location=.Editor --extra-editor-arg="-colorspace=Linear" --extra-editor-arg="-force-glcore" --platform=editmode --scripting-backend=Mono2x --suite=editor --testfilter=$TEST_FILTER --testproject=./TestProjects/ShaderGraph --zero-tests-are-ok=1
+    after:
+      -  command: python .yamato/script/log_scripts/log_parser.py
```

Comment:
And for some jobs there is no `pip install requests` command. I'm guessing it's all good because some bokken images already have it installed, but just a sanity check remark :)

Reply:
Yep exactly :) I will double-check though, thanks!